File: src/IntegrationApp/IntegrationAPI/DTOs/TenderOfferDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Integration_API.DTOs
{
    public class TenderOfferDto
    {
        public string DrugstoreName { get; set; }
        public List<DrugTenderDto> TenderInfo { get; set; }
        public int Price { get; set; }
        public string TenderId { get; set; }

        public TenderOfferDto()
        {
            TenderInfo = new List<DrugTenderDto>();
        }

        public TenderOfferDto(string drugstoreName ,List<DrugTenderDto> tenderInfo, int price, string tenderId)
        {
            DrugstoreName = drugstoreName;
            TenderInfo = tenderInfo;
            Price = price;
            TenderId = tenderId;
        }
    }
}

File: src/ManagerAngularApp/ManagerAngularApp/src/app/drugstore-chart/tender-offer.dto.ts
import { TenderDto } from "../tender/tender.dto";

export class TenderOfferDto{

    drugstoreName: string;
    tenderInfo: TenderDto[];
    price: number;

    constructor(){
        this.drugstoreName = "";
        this.tenderInfo = [];
        this.price = 0;
        
    }
}